psh/test-date.py

```diff
@@ -0,0 +1,154 @@
+# Phoenix-RTOS
+#
+# phoenix-rtos-tests
+#
+# psh date command test
+#
+# Copyright 2022, 2023 Phoenix Systems
+# Author: Mateusz Niewiadomski, Mateusz Bloch
+#
+# This file is part of Phoenix-RTOS.
+#
+# %LICENSE%
+#
+
+import psh.tools.psh as psh
+
+DFLT_DATE_REG = r"\w{3}, \d{2} \w{3} \d{2} \d{2}:\d{2}:\d{2}\r+\n"  # default date when image is flashed
```

Comment:
After consideration I think we should also pare such output (year can be printed using four digits, the comma after year and info about time zone are possible). Comment not valid now.

```suggestion
DFLT_DATE_REG = r"\w{3}, \d{2} \w{3} \d{2} \d{2}:\d{2}:\d{2}\r+\n"
```